### ISSUE TITLE
Training success time?

### ISSUE THREAD

**Issue by Liuzy0908** [ValentinaZangirolami/DRL#2]
This is an amazing project. I have successfully run this project. 

I'd like to ask you some questions here. The action of the agent has not improved after I run it for some hours, so I would like to know  how long did it take for the agent to complete a lap? Further, can you provide some trained parameter file?

yours sincerely

**Comment by ValentinaZangirolami** [comment]
> This is an amazing project. I have successfully run this project.
> 
> I'd like to ask you some questions here. The action of the agent has not improved after I run it for some hours, so I would like to know how long did it take for the agent to complete a lap? Further, can you provide some trained parameter file?
> 
> yours sincerely

During the training time, you may notice agent's improvement after a long time. However, it even depends on the choice of exploration strategy. In our study,  we observed that, in general, after 1 milion of frames we get good results in both training set and test set. This number of frames corresponds to about 1/2 days, but it also depends on your hardware capability. 

I can provide you a trained parameter file, contact me at valentina.zangirolami@gmail.com . 

Thanks
